libbeat/tests/system/beat/beat.py

```diff
@@ -180,7 +180,8 @@ def start_beat(self,
                    output=None,
                    logging_args=["-e", "-v", "-d", "*"],
                    extra_args=[],
-                   env={}):
+                   env={},
+                   configure_home=True):
```

Comment:
As it seems that home has to be configured in any case, wdyt about adding home as argument, and if it is set then it is used, and if not, the current default is used?
```suggestion
                   home=""):
```
...
```
if not home:
    home = os.path.normpath(self.working_dir)

args += [
    "-path.home", home,
   ...
```

Reply:
Although, it's not needed anymore I refactored it anyway. It seems more fitting that home is configurable.

Reply:
Great, thanks